Docs/install/index.rst

```diff
@@ -34,118 +34,193 @@
    #  @@-COPYRIGHT-END-@@
    # =============================================================================
 
+.. role:: hideitem
+   :class: hideitem
+
 .. _ug-installation:
 
-###################
+##################
 AIMET Installation
-###################
+##################
 
-Quick Install
-~~~~~~~~~~~~~
+This page contains instructions for the following installation scenarios:
+
+- :ref:`Quick Install of the latest version for PyTorch <idx-install-quick>`
+- :ref:`Installation of the latest version for all platforms <idx-install-latest>`
+- :ref:`Installation of older versions <idx-install-older>`
+
+You can also:
 
-The AIMET PyTorch GPU PyPI packages are available for environments that meet the following requirements:
+- :doc:`Install manually on a host machine <install_host>`
+- :doc:`Install a pre-built or locally built image in a Docker container <install_docker>`
+
+.. note::
+    
+    The following system requirements apply to all installations.
+
+System requirements
+===================
+
+The AIMET package requires the following host platform setup:
 
 * 64-bit Intel x86-compatible processor
-* Linux Ubuntu 22.04 LTS [Python 3.10] or Ubuntu 20.04 LTS [Python 3.8]
+* Linux Ubuntu 22.04 LTS (Python 3.10) or Ubuntu 20.04 LTS (Python 3.8)
+* bash command shell
+* For GPU variants:
+    * Nvidia GPU card (Compute capability 5.2 or later)
+    * Nvidia driver version 455 or later (using the latest driver is recommended; both CUDA and cuDNN are supported)
+    * nvidia-docker - Installation instructions: https://github.com/NVIDIA/nvidia-docker
+
+.. _idx-install-quick:
+
+Quick Install
+=============
+
+The fastest way to install AIMET is to use the AIMET PyTorch GPU PyPI packages.
+
+Prerequisites
+-------------
+
+The following software versions are required for the quick install:
+
 * CUDA 12.0
 * Torch 2.2.2
 
-**Pip install**
+Ensure that you have the LAPACK linear algebra and libpython3-dev development packages installed:

```

Comment:
Can we replace "LAPACK linear algebra and libpython3-dev development" with just "pre-requisite" packages? The package names are quite clear from the code block, and it will be succinct and easier to maintain in future.

Reply:
Done.